architecture/core_concepts/builds_and_image_streams.adoc

```diff
@@ -540,6 +540,39 @@ Once the secret is configured proceed with creating new Image Stream or using
 `oc import-image` command. During the import process OpenShift will pick up the
 secrets and provide to the remote party.
 
+[[crossnamespace-image-import]]
+
+=== Importing Images from internal registry
+
+An image stream can be configured to import tag and image metadata from internal
+registry but from a different project. The recommended approach for this, is to use
+link:#adding-tag[`*oc tag*`] command:
+
+====
+----
+$ oc tag originalproject/imagestream:tag newimagestream:newtag
+----
+====
+
+Other option, to import image from internal registry, from other project manually,
```

Comment:
I'd strongly deter from following this scenario. I'd stress that this should be avoided whenever possible.

It could be even moved to a troubleshoot section:

> Getting *unathorized* while trying to import image from an internal registry?

> Don't do that! Or if you need to, do the policy trick.